src/Provider/Facebook.php

```diff
@@ -104,17 +104,23 @@ public function getDefaultScopes(): array
         return ['public_profile', 'email'];
     }
 
-    public function getResourceOwnerDetailsUrl(AccessToken $token): string
+    public function getResourceOwnerDetailsUrl(AccessToken $token, array $fields = null): string
```

Comment:
This modifies the signature of [AbstractProvider::getResourceOwnerDetailsUrl][1]. Instead, I suggest that `fields` be added as a parameter to the constructor `$options`.

[1]: https://github.com/thephpleague/oauth2-client/blob/2.6.0/src/Provider/AbstractProvider.php